docs/tutorials/local_iteration_series/local-Iteration-testing-multiplayer-games-locally.md

```diff
@@ -0,0 +1,62 @@
+---
+id: local_iteration_testing_locally
+title: Local Iteration - Testing multiplayer games locally
+description: Guide covering the available workflows for testing multiplayer games locally.
+---
+
```

Comment:
It'd be great to have a TOC with a quick summary. What's the one exam question you'd like to ask your readers?